### ISSUE TITLE
[ADE Support]: Accredited Representation Management: Representative Status in Profile: 508 Audit Report Findings - Defect 1

### ISSUE THREAD

**Issue by lakper** [department-of-veterans-affairs/va.gov-team#112267]
> [!NOTE]
> **QUICK SUMMARY:** _Child ticket of #112265 tracking defect 1._

### Defect 1 | Critical | Created:
**_WCAG 1.1.1 Non-text Content | All non-text content that is presented to the user has a text alternative that serves the equivalent purpose, except for the situations listed below._**

On the Get Help From A VA Accredited Representative Or VSO | Veterans Affairs (No Representative) screen (Get help from an accredited representative or VSO), a decorative image is read by assistive technology—“Your representative Graphic” is read under Check if you already have an accredited representative. (New: 06/16/2025)

Project Site: [staging.va.gov/profile/accredited-representative](http://staging.va.gov/profile/accredited-representative)

### Ticket Updates
- [x] _Connect to an `Epic/Parent` or `Intake` (what body of work is this a part of?) If this is the `Epic/Parent`, connect related tickets as sub-issues._
- [ ] _If this is the `Epic/Parent` ticket, add label for `ADE 508 RECHECK`_
- [x] _If this ticket is a `Child` ticket aka `sub-issue` please add the appropriate labels for `ADE 508 FLAG CRITICAL`;`ADE 508 FLAG HIGH`; `ADE 508 FLAG MEDIUM` or `ADE 508 FLAG LOW`_
- [x] _Label with `Originator/Team` (product team or stakeholder requesting support; ADE if internal)_
- [x] _Label date in the `Open Date` field_
- [ ] _Label with `Estimate` (level of effort expected for this ticket)_
- [x] _Add `Assignee(s)` name(s) to ticket_ (include Lakwi)
- [ ] _Add `Assignee(s) name(s) to each task` they will complete via handle tag (if known)_
- [ ] _Select a `Priority Level`_
- [x] _Update date in `Last Checked` field_
- [ ] _Label with `Actual` (level of effort it took to complete this ticket)_
- [ ] _Update date in_ `Closed Date`
- [ ] _Make sure the `ADE 508 RECHECK` label is on this ticket upon closing and that a follow-up date has been established and calendared._

**Comment by coforma-jamie** [comment]
I believe he's talking about the person icon that appears next to the representative's name in the widget ([this page](https://staging.va.gov/get-help-from-accredited-representative/#check-if-you-already-have-an-a), for logged-in users with a rep).

Assuming that's the case, **I'm not able to duplicate the issue.** The icon uses `<va-icon>`, in the shadow DOM the svg has `aria-hidden="true"` which hides it from assistive technology. And using JAWS, as I arrow around, I'm unable to select the icon and no text description is read as a result: [Clip](https://coformaco.zoom.us/clips/share/xmOU5MsATOqW0oGBzJPhtg)
